### ISSUE TITLE
AssertionError The command '/bin/sh -c python3 -m pip install -e . --no-cache-dir' returned a non-zero code: 2

### ISSUE THREAD

**Issue by benalb** [xserrat/docker-facebook-demucs#35]
### Contact Details

benalb@gmail.com

### What happened?

After cloning the repo, both trying "make build" or "make run track=test01.mp3" gives the same AssertionError

### In which operating system you found the issue?

Linux

### Relevant log output

```shell
Successfully built julius dora-search antlr4-python3-runtime treetable
ERROR: Exception:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/pip/_internal/cli/base_command.py", line 165, in exc_logging_wrapper
    status = run_func(*args)
  File "/usr/lib/python3/dist-packages/pip/_internal/cli/req_command.py", line 205, in wrapper
    return func(self, options, args)
  File "/usr/lib/python3/dist-packages/pip/_internal/commands/install.py", line 389, in run
    to_install = resolver.get_installation_order(requirement_set)
  File "/usr/lib/python3/dist-packages/pip/_internal/resolution/resolvelib/resolver.py", line 188, in get_installation_order
    weights = get_topological_weights(
  File "/usr/lib/python3/dist-packages/pip/_internal/resolution/resolvelib/resolver.py", line 276, in get_topological_weights
    assert len(weights) == expected_node_count
AssertionError
The command '/bin/sh -c python3 -m pip install -e . --no-cache-dir' returned a non-zero code: 2
make: *** [Makefile:60: build] Error 2
```

**Comment by xserrat** [comment]
Hi @benalb,

Thanks for reporting the issue. I've just merged the fix #36 forcing a specific version of the Demucs lib to keep the execution stable.

Try with the version `v1.0.0` of this repository to make sure that the issue has been solved.

Thanks in advance.

**Comment by benalb** [comment]
wow, that was fast :)
works ok, thank you very much. 
I'm closing the issue.
